docs/resources/chain-integration.md

```diff
@@ -60,12 +56,21 @@ To provide a stable experience to users, there is a request rate limit on the pu
 | ELON       | [0x02DCcaf514C98451320a9365C5b46C61d3246ff3](https://cronos.crypto.org/explorer/address/0x02DCcaf514C98451320a9365C5b46C61d3246ff3/transactions)   | 18      |
 | TUSD       | [0x87EFB3ec1576Dec8ED47e58B832bEdCd86eE186e](https://cronos.crypto.org/explorer/address/0x87EFB3ec1576Dec8ED47e58B832bEdCd86eE186e/transactions)   | 18      |
 
-## API Clients and libraries
+## Integration guide for **Cronos Testnet**
+### Setup Guide
 
-- [**TypeScript** library](https://github.com/crypto-org-chain/chain-jslib);
-- [**Python** library](https://pypi.org/project/chainlibpy/#description);
-- [**Rust** library](https://github.com/crypto-org-chain/chainlib-rs) (note that it is not feature complete);
-- [@cosmjs/stargate](https://github.com/cosmos/cosmjs/tree/master/packages/stargate).
+- [Joining the Cronos Testnet](https://cronos.crypto.org/docs/getting-started/cronos-testnet.html)
+- [Testnet Faucet](https://cronos.crypto.org/faucet/)
+  - Note that the faucet only takes eth-type `0x...` addr
+- [Seeds for fullnode](https://github.com/crypto-org-chain/cronos-testnets/blob/main/testnet.json#L21)
+- [Genesis files](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet_338-1/genesis.json)
```

Comment:
Very minor suggestion: Just to align with the mainnet one and easier for download/`cp`. 
```suggestion
- [Genesis files](https://raw.githubusercontent.com/crypto-org-chain/cronos-testnets/main/cronostestnet_338-1/genesis.json)
```